react/index.js

```diff
@@ -37,11 +38,13 @@ class AddressManager extends Component {
     }
   }
 
+  cleanRedirectUrl = (returnURL) => head(returnURL) === '/' ? returnURL : `/${returnURL || ''}`
+
   redirectToReturnURL = () => {
     try {
       const parsedQueryString = queryString.parse(window.location.search)
-      const returnURL = parsedQueryString && parsedQueryString.returnUrl
-      window.location.href = `/${returnURL || ''}`
+      const returnURL = parsedQueryString && parsedQueryString.returnUrl  
+      window.location.href = this.cleanRedirectUrl(returnURL)
```

Comment:
You could just
```suggestion
      window.location.href = head(returnURL) === '/' ? returnURL : `/${returnURL || ''}`
```
Instead of creating a function...
(unless you plan to use it somewhere else)